main/src/modules/attentionRelated/ears/src/ears.cpp

```diff
@@ -26,6 +28,36 @@ bool ears::configure(yarp::os::ResourceFinder &rf)
     MainGrammar = rf.findFileByName(rf.check("MainGrammar", Value("MainGrammar.xml")).toString());
     bShouldListen = true;
 
+    if (followPlans){
+        /*portToBehavior.open("/" + moduleName + "/behavior:o");
+        while (!Network::connect(portToBehavior.getName(),"/GoalManager/trigger:i ")) {
+            yWarning() << " Behavior is not reachable";
+            yarp::os::Time::delay(0.5);
+        }*/
+
+        // port to planner module
+        port_planner.open("/" + moduleName + "/target:o");
+        while (!Network::connect(port_planner.getName(),"/planner/rpc"))
+        {
+            yWarning() << "Planner is unreachable...";
+            yarp::os::Time::delay(0.5);
+        }
+
+        goal_need_port.open("/" + moduleName + "/goal:o");
```

Comment:
Where is the connection for this port made? It should be in an XML, but I cannot find it anywhere.